pkgs/applications/audio/nuclear/default.nix

```diff
@@ -21,7 +20,7 @@ in appimageTools.wrapType2 {
       --replace 'Exec=AppRun' 'Exec=${pname}'
     cp -r ${appimageContents}/usr/share/icons $out/share
   '';
-
+  extraPkgs = pkgs: with pkgs; [ wrapGAppsHook ];
```

Comment:
this won't be needed once https://github.com/NixOS/nixpkgs/pull/161739 is merged